hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/main/java/org/hawkular/alerts/actions/email/PluginMessageDescription.java

```diff
@@ -477,7 +476,7 @@ public String compare(CompareCondition condition) {
      * @return a description to be used on email templates
      */
     public String external(ExternalCondition condition) {
-        String description = "SystemId: " + condition.getSystemId();
+        String description = "AlerterId: " + condition.getAlerterId();
```

Comment:
This is just a renaming for clarity.  I had a TODO for this and decided to do it as part of reviving the metrics external alerter.  So, no need to worry too much about this refactor.